Kiosk/App/AppDelegate+GlobalActions.swift

```diff
@@ -129,39 +161,53 @@ private extension AppDelegate {
 
     // MARK: - s that do things
 
-    func ツ() -> Observable<Void>{
+    func ツ() -> Observable<Void> {
         return hideAllTheThings()
     }
 
     func hideAllTheThings() -> Observable<Void> {
         return self.closeFulfillmentViewController().then(self.hideHelp())
     }
-    
+
     func showBidderDetailsRetrieval() -> Observable<Void> {
         let appVC = self.appViewController
-        let presentingViewController: UIViewController = (appVC!.presentedViewController ?? appVC!)
-        return presentingViewController.promptForBidderDetailsRetrieval(provider: self.provider)
+        let presentingViewController: UIViewController = (
+            appVC!.presentedViewController ?? appVC!
+        )
+        return presentingViewController.promptForBidderDetailsRetrieval(
+            provider: self.provider
+        )
     }
 
     func showRegistration() -> Observable<Void> {
         return Observable.create { observer in
             ARAnalytics.event("Register To Bid Tapped")
 
             let storyboard = UIStoryboard.fulfillment()
-            let containerController = storyboard.instantiateInitialViewController() as! FulfillmentContainerViewController
+            let containerController =
+                storyboard.instantiateInitialViewController() as! FulfillmentContainerViewController
             containerController.allowAnimations = self.appViewController.allowAnimations
 
-            if let internalNav: FulfillmentNavigationController = containerController.internalNavigationController() {
+            if
+                let internalNav: FulfillmentNavigationController =
```

Comment:
feels like `if let` is something that should probably be kept together unless there's many `let`s in the `if` expression